apps/passport-server/test/generic-issuance/pipelines/lemonade/setupLemonadePipeline.ts

```diff
@@ -285,6 +243,13 @@ export function setupLemonadePipeline(): LemonadePipelineTestData {
     type: PipelineType.Lemonade
   };
 
+  const mockServer = setupServer(
+    ...getMockLemonadeHandlers(lemonadeBackend, lemonadeBackendUrl)
+  );
+  // The mock server will intercept any requests for URLs that are registered
+  // with it. Unhandled requests will bypass the mock server.
+  mockServer.listen({ onUnhandledRequest: "bypass" });
```

Comment:
This function is called from the top-level `describe()` in the test file. This means that the mock server will start listening at the point where the tests are loaded. Since other tests may run first, the listener will be active during tests other than those for the Lemonade pipeline. This is the cause of test failures in the Devconnect tests, because the Lemonade mock server does not have the correct handlers. Since I was assigned the bug for this, I made a PR here https://github.com/proofcarryingdata/zupass/pull/1603, so you could either merge that or fix it separately and delete the PR.

Reply:
thanks so much!